python/tvm/driver/build_module.py

```diff
@@ -231,6 +231,8 @@ def build(
     elif isinstance(inputs, PrimFunc):
         input_mod = lower(inputs, name=name)
     elif isinstance(inputs, tvm.IRModule):
+        if name is not None and name != "default_function":
```

Comment:
Let’s skip this - likely it’s a mistake during rebase

Reply:
See https://github.com/apache/tvm/pull/9678

Reply:
Thanks for the reminder! Just reverted the change.